api/lib/stores/block-store.js

```diff
@@ -264,6 +266,68 @@ class BlockStore {
       'operatorAddress'
     )
   }
+
+  async updateNetworkInDB() {
+    try {
+      const storedNetwork = await database(config)('').getNetwork(
+        this.network.id
+      )
+      if (storedNetwork) {
+        // we update everything except the network id and powered
```

Comment:
why are we not updating those 2? why are we so explicit about all the values? can't we just do ```Object.assign(this.network, storedNetwork)```?

Reply:
powered should come from the DB, right?

Reply:
if we can update networkId, many things would fail. Static things such as the themes in FE. It is not a good idea to do it. Also the messageCreator. So it would be impossible to create transactions. This is based on file names and would get automatically broken

Reply:
About powered, yes, we could do it. It is just a pain in the ass to edit it because it is in this format: `"{&quot;name&quot;:&quot;Forbole&quot;,&quot;providerAddress&quot;:&quot;kavavaloper14kn0kk33szpwus9nh8n87fjel8djx0y02c7me3&quot;,&quot;picture&quot;:&quot;https://lunie.fra1.digitaloceanspaces.com/validator-pictures/forbole.jpeg&quot;}"` and you need to be super careful with characters here. One little character extra and gone after editing would mean deserialization broken and error

Reply:
We can seat and we can talk why this escape thing seems necessary. Perhaps there is a workaround, but I can confirm that directly sending stringified JSONs won't work (it is a quote `"` inside `"` problem, but then others quotes (` ') won't be parsed like in JS)

Reply:
we could remove powered and id with `delete` and then we could do Object.assign. Or we could also have another table just for powered.

Reply:
Waiting on instructions :v: